tests/run-pass/backtrace-api-v0.stderr

```diff
@@ -2,17 +2,4 @@ $DIR/backtrace-api-v0.rs:LL:CC (func_d)
 $DIR/backtrace-api-v0.rs:LL:CC (func_c)
 $DIR/backtrace-api-v0.rs:LL:CC (func_b)
 $DIR/backtrace-api-v0.rs:LL:CC (func_a)
-$DIR/backtrace-api-v0.rs:LL:CC (main)
-rustc_src/src/ops/function.rs:LL:CC (<fn() as std::ops::FnOnce<()>>::call_once - shim(fn()))
-rustc_src/src/sys_common/backtrace.rs:LL:CC (std::sys_common::backtrace::__rust_begin_short_backtrace)
-rustc_src/src/rt.rs:LL:CC (std::rt::lang_start::{closure#0})
-rustc_src/src/ops/function.rs:LL:CC (std::ops::function::impls::call_once)
-rustc_src/src/panicking.rs:LL:CC (std::panicking::r#try::do_call)
-rustc_src/src/panicking.rs:LL:CC (std::panicking::r#try)
-rustc_src/src/panic.rs:LL:CC (std::panic::catch_unwind)
-rustc_src/src/rt.rs:LL:CC (std::rt::lang_start_internal::{closure#2})
-rustc_src/src/panicking.rs:LL:CC (std::panicking::r#try::do_call)
-rustc_src/src/panicking.rs:LL:CC (std::panicking::r#try)
-rustc_src/src/panic.rs:LL:CC (std::panic::catch_unwind)
-rustc_src/src/rt.rs:LL:CC (std::rt::lang_start_internal)
-rustc_src/src/rt.rs:LL:CC (std::rt::lang_start)
```

Comment:
I don't know why all these lines are disappearing now.  Or does this show that some kind of backtrace pruning that was broken is working again after this PR?